src/pollution_model/run_medusa_2.py

```diff
@@ -622,12 +625,69 @@ def main(selected_polygon_gdf: gpd.GeoDataFrame,
     rainfall_event = MedusaRainfallEvent(antecedent_dry_days, average_rain_intensity, event_duration, rainfall_ph)
 
     # Run the pollution model
-    scenario_id = run_pollution_model_rain_event(engine, area_of_interest, rainfall_event)
+    scenario_id = run_pollution_model_rain_event(
+        engine, area_of_interest, rainfall_event
+    )  # pylint: disable=unsupported-assignment-operation
+
     # Ensure pollution model data is being served by geoserver
     serve_pollution_model()
+
+    # Create new table recording users' history
+    create_table(engine, MedusaScenarios)
+    # Create the query object
+    query = MedusaScenarios(
+        scenario_id=scenario_id,
+        antecedent_dry_days=antecedent_dry_days,
+        average_rain_intensity=average_rain_intensity,
+        event_duration=event_duration,
+        rainfall_ph=rainfall_ph,
+        geometry=area_of_interest['geometry'].to_wkt().iloc[0]
+    )
+    # Execute the query
+    execute_query(engine, query)
+
     return scenario_id
 
 
+def retrieve_input_parameters(scenario_id: int) -> Dict[str, Union[str, float]]:
+    # pylint: disable=unsupported-assignment-operation
```

Comment:
```suggestion
```
It's not recommended to disable this for the entire function, its a useful warning that we should not fully ignore.
Let me know if this causes annoyance in programming though, because we can reconsider